src/azure-cli/azure/cli/command_modules/cosmosdb/_params.py

```diff
@@ -53,46 +53,77 @@ class ThroughputTypes(str, Enum):
 
 def load_arguments(self, _):
     from knack.arguments import CLIArgumentType
-    from azure.mgmt.cosmosdb.models import KeyKind, DefaultConsistencyLevel, DatabaseAccountKind, TriggerType, TriggerOperation, ServerVersion, NetworkAclBypass
+    from azure.mgmt.cosmosdb.models import KeyKind, DefaultConsistencyLevel, DatabaseAccountKind, TriggerType, TriggerOperation, ServerVersion, NetworkAclBypass, BackupPolicyType
 
     with self.argument_context('cosmosdb') as c:
-        c.argument('account_name', arg_type=name_type, help='Name of the Cosmos DB database account', completer=get_resource_name_completion_list('Microsoft.DocumentDb/databaseAccounts'), id_part='name')
-        c.argument('database_id', options_list=['--db-name', '-d'], help='Database Name')
+        c.argument('account_name', arg_type=name_type, help='Name of the Cosmos DB database account',
+                   completer=get_resource_name_completion_list('Microsoft.DocumentDb/databaseAccounts'), id_part='name')
+        c.argument('database_id', options_list=[
+                   '--db-name', '-d'], help='Database Name')
 
     with self.argument_context('cosmosdb create') as c:
         c.argument('account_name', completer=None)
         c.argument('key_uri', help="The URI of the key vault", is_preview=True)
-        c.argument('enable_free_tier', arg_type=get_three_state_flag(), help="If enabled the account is free-tier.", is_preview=True)
-        c.argument('assign_identity', nargs='*', help="accept system or user assigned identities separated by spaces. Use '[system]' to refer system assigned identity. Currently only system assigned identity is supported.", is_preview=True)
+        c.argument('enable_free_tier', arg_type=get_three_state_flag(),
+                   help="If enabled the account is free-tier.", is_preview=True)
+        c.argument('assign_identity', nargs='*',
+                   help="accept system or user assigned identities separated by spaces. Use '[system]' to refer system assigned identity. Currently only system assigned identity is supported.", is_preview=True)
+        c.argument('is_restore_request', options_list=['--is-restore-request', '-r'], arg_type=get_three_state_flag(
+        ), help="Restore from an existing/deleted account.", is_preview=True, arg_group='Restore')
```

Comment:
```suggestion
        c.argument('is_restore_request', options_list=['--is-restore-request', '-r'], arg_type=get_three_state_flag(),
        help="Restore from an existing/deleted account.", is_preview=True, arg_group='Restore')
```

Reply:
was this comment related to formatting? i fixed the formatting.